Public/peep.css

```diff
@@ -0,0 +1,145 @@
+body {
```

Comment:
Can't wait to see this CSS in the demo. I'd be interested to know where you learnt your CSS?